addon/resolver.js

```diff
@@ -25,6 +25,19 @@ import EmberResolver from 'ember-resolver';
 */
 export default EmberResolver.extend({
   /**
+    Contains names of type for which need lookup class with prefix received through a {{#crossLink "DeviceService"}}{{/crossLink}}.
+
+    @property resolvingTypes
+    @type Array
+    @default ['component', 'template', 'view']
+  */
+  resolvingTypes: [
```

Comment:
Array contains types that must be resolved relative to the current device type, but property name 'resolvingTypes' makes a sense, that it is something global for all types. Maybe something like 'deviceRelatedTypes' will be more correct.